workflows/execute_workflows.sh

```diff
@@ -53,6 +53,25 @@ git -C submodules/oton submodule update --init
 echo "Update quiver-data if necessary …"
 git -C submodules/quiver-data pull origin main
 
+echo "Installing OtoN …"
+pushd submodules/oton
+pip install .
+popd
+echo "OtoN installation done."
+
+echo "Installing quiver-ocrd"
+pip install .
+echo "quiver-ocrd installation done."
+
+quiver-ocrd || {
```

Comment:
```suggestion
which quiver-ocrd || {
```

should be enough. Otherwise, I'd do `x --version` or `x --help` because no-arg calls might result in non-zero exit code.